airbyte-server/src/main/java/io/airbyte/server/apis/LogsApiController.java

```diff
@@ -0,0 +1,17 @@
+package io.airbyte.server.apis;
+
+import io.airbyte.api.generated.LogsApi;
+import io.airbyte.api.model.generated.LogsRequestBody;
+import io.airbyte.server.handlers.LogsHandler;
+import java.io.File;
+import lombok.AllArgsConstructor;
+
+@AllArgsConstructor
+public class LogsApiController implements LogsApi {
```

Comment:
Is this missing a path?

Reply:
Yes and no. Adding the path is not needed because it is inherited from the interface but I prefer to add it in order to avoid to have to look at the interface know the path when you are debugging. I have added it now.